manager/ui/war/plugins/api-manager/html/api/api-plans.html

```diff
@@ -46,14 +46,14 @@
             <div class="title"
                  apiman-i18n-key="available-plans">Available Plans</div>
             <div>
-              <span apiman-i18n-key="available-plans-help">Choose which plans should be presented when Client Apps create a link (Contract) to this API. Note that only plans in a 'Locked' state show up in this list.</span>
+              <span apiman-i18n-key="available-plans-help">Choose which plans should be presented when Client Apps create a contract to this API. Only plans in a Locked state show up in this list..</span>

```

Comment:
Change locked to lower case and use `<strong>` or `<em>` if you are not going to use quotations.